packages/server/lib/errors.coffee

```diff
@@ -106,9 +106,19 @@ getMsgByType = (type, arg1 = {}, arg2, arg3) ->
       str = """
       Can't run because you've entered an invalid browser name.
 
-      Browser: '#{arg1}' was not found on your system.
+      Browser: '#{arg1}' was not found on your system or is not supported by Cypress.
 
-      Available browsers found are: #{arg2}
+      Cypress supports the following browsers:
+      - chrome
+      - chromium
+      - edge
+      - electron
+      - firefox (Cypress support in beta)
+
+      You can also use a custom browser: https://on.cypress.io/launching-browsers#Customize-available-browsers
```

Comment:
We will need a custom on link for this.

Reply:
https://github.com/cypress-io/cypress-services/pull/2601